sklearn/model_selection/tests/test_split.py

```diff
@@ -1376,17 +1388,22 @@ def test_group_kfold():
     # Should fail if there are more folds than groups
     groups = np.array([1, 1, 1, 2, 2])
     X = y = np.ones(len(groups))
-    assert_raises_regexp(ValueError, "Cannot have number of splits.*greater",
-                         next, GroupKFold(n_splits=3).split(X, y, groups))
+    with pytest.raises(
+                            ValueError,
+                            match="Cannot have number of splits.*greater"
+                        ):
+        next(GroupKFold(n_splits=3).split(X, y, groups))
```

Comment:
nitpick, I think the future indenting style we are converging to for the scikit-learn project for such multiline expressions would rather be the following (@glemaitre correct me if I am wrong):

```suggestion
    with pytest.raises(
        ValueError,
        match="Cannot have number of splits.*greater"
    ):
        next(GroupKFold(n_splits=3).split(X, y, groups))
```

Reply:
It looks correct :)